core/src/main/java/com/novoda/merlin/service/HostPinger.java

```diff
@@ -21,29 +16,22 @@
 
     }
 
-    static HostPinger withDefaultEndpointValidation(PingerCallback pingerCallback) {
```

Comment:
Default is now specified in the `MerlinBuilder` rather than defaulting to this further down.